docs/adr/0011-message-timing-information.md

```diff
@@ -0,0 +1,75 @@
+# 11. Message Timing Information
+
+Date: 2019-07-05
+
+## Status
+
+Accepted
+
+## Context
+
+We need to decide whether message timing information should be exposed via the
+API. Timing information refers to important points in time throughout the
+lifecycle of a message.
+
+The initial rationale for *not* exposing these timestamps was that any business
+logic that depends on time in some way should include any time related
```

Comment:
should `time related` be hyphenated here?

Reply:
I'll revisit this terminology throughout the ADR, it's probably a bit inconsistent atm.